config.template.yml

```diff
@@ -516,3 +516,16 @@ notifier:
   ##   sender: admin@example.com
   ##   host: smtp.gmail.com
   ##   port: 587
+
+identity_providers:
+  oidc:
+    hmac_secret: this_is_a_secret_abc123abc123abc
```

Comment:
We should probably put some comments to document what it is used for.

Reply:
Done!

Reply:
That's exactly what it's meant for. The wording looks good to me.